src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -37,6 +37,30 @@ import { kTexelRepresentationInfo } from '../../util/texture/texel_data.js';
 
 import { ValidationTest } from './validation_test.js';
 
+const kBlendFactors: GPUBlendFactor[] = [
```

Comment:
nit: I think this could go in capability_info.ts along with a TS-doc description.